### ISSUE TITLE
Parameterized Constructor does not allow library disambiguation

### ISSUE THREAD

**Issue by bclothier** [WaynePhillipsEA/twinbasic#412]
**Describe the bug**
When newing up an object using a parameterized constructor, we get an error when we explicitly specify the library.

**To Reproduce**
Assume the project is named "Sample", given this code:
```
Public Class Example
    Public Sub New()
    End Sub
    
    Public Sub New(Foo As String)
    End Sub
End Class

Public Module Test
    Public Sub DoIt()
        Dim x1 As Example
        Dim x2 As Example
        Set x1 = New Sample.Example
        Set x2 = New Sample.Example("foo")
    End Sub
End Module
```

The `Set x1` assignment has no problem but the `Set x2` assignment has a red squiggly line under the `Sample` indicating that "[Sample] is a project or library, not a datatype".

**Expected behavior**
It should be always legal to disambiguate, even for the parameterized construction. 

**Desktop (please complete the following information):**
 - OS: Windows 7
 - twinBASIC compiler version v0.10.5646

**Comment by WaynePhillipsEA** [comment]
Thanks, I'll get that fixed.

**Comment by WaynePhillipsEA** [comment]
Fixed in 0.10.5711, thanks for reporting
